web/crux/src/app/container/container.mapper.ts

```diff
@@ -72,7 +72,8 @@ export default class ContainerMapper {
       expose: config.expose,
       ingress: toPrismaJson(config.ingress),
       configContainer: toPrismaJson(config.configContainer),
-      user: config.user ? config.user : null,
+      // Set user to the given value, if not null or use 0 if specifically 0, otherwise set to default: -1
+      user: config.user || config.user === 0 ? (config.user === 0 ? 0 : config.user) : -1,
```

Comment:
This can be simplified to: `config.user || config.user === 0 ? config.user : -1`

Reply:
This is not the same.